lib/cloudshaper/stack.rb

```diff
@@ -29,6 +29,8 @@ def initialize(config)
       @stack_dir = File.join(Stacks.dir, @stack_id)
 
       @module = StackModules.get(config.fetch('root'))
+      @module.build(cloudshaper_stack_id: @stack_id)
```

Comment:
Environment vars keeps things consistent. I checked it out, and environment vars only work for the root module, but I found a way to propagate this downwards through submodules (see new commit on this PR).